docs/config/examples.rst

```diff
@@ -147,6 +147,35 @@ in the UMF repository.
 
 TODO
 
+CTL statistics example
+==============================================================================
+
```

Comment:
pls mentione similarly to above examples - `... rely on experimental APIs that may change in future releases.`

Reply:
done